samples/complex_mig_default_network/terraform.tfvars

```diff
@@ -0,0 +1,26 @@
+project_id      = "project-id"
+resource_prefix = "complex-mig1"
+zone            = "us-central1-a"
+machine_type    = "a2-highgpu-1g"
+target_size     = 1
+guest_accelerator = {
+  count = 1
+  type  = "nvidia-tesla-a100"
+}
```

Comment:
I thought that guest_accelerator had to be null when using machine_types a2 and g2 since their accelerators are defined by the machine type